docs/reference/controllers.md

```diff
@@ -138,6 +138,8 @@ import ReferenceController from "./controllers/reference_controller"
 application.register("reference", ReferenceController)
 ```
 
+Use kebab-case in references to ensure features like targets function correctly (i.e., controllers/date_picker_controller, DatePickerController, date-picker)
```

Comment:
What do you think, would it instead make sense to update the example above to include the kebab-case version?